components/sidebar/sidebar.tsx

```diff
@@ -122,11 +124,19 @@ export default class Sidebar extends React.PureComponent<Props, State> {
     }
 
     render() {
+        const {
```

Comment:
Can you also revert these changes? We have a lot of upcoming changes in the sidebar feature branch, so I'd like to avoid conflicts when possible.